src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -95,6 +95,18 @@
     <Compile Include="..\..\src\Microsoft\Data\Common\AdapterUtil.Windows.cs">
       <Link>Microsoft\Data\Common\AdapterUtil.Windows.cs</Link>
     </Compile>
+    <Compile Include="..\..\src\Microsoft\Data\Common\DbBatch.cs">
```

Comment:
IIRC, even though we are not implementing/exposing the new batch APIs in netfx or netstd, we need these included in the project in order to share code with netcore, right?

Reply:
Yes.